tests/unit/service/test_infrastructure.py

```diff
@@ -196,6 +217,12 @@ def test_find_missing_instance_name(self, logging_context):
 
 class TestInfrastructureService(unittest.TestCase):
 
+    def __propvaluemap(self, orig_props):
+        props = {}
+        for k, v in orig_props.items():
+            props[k] = {'type': 'string', 'value': v}
+        return PropValueMap(props)
```

Comment:
picky: couldn't you use a dictionary comprehension here to avoid the explicit construction of a variable?